keyboards/crkbd/keymaps/thunderbird2086/process_records.c

```diff
@@ -23,18 +24,32 @@ bool process_record_oled(uint16_t keycode, keyrecord_t *record) {
 #endif
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
-    switch (keycode) {
-
-    case COLEMAK:
-        if (record->event.pressed) {
-            set_single_persistent_default_layer(_COLEMAK);
-        }
-        break;
-    case QWERTY:
-        if (record->event.pressed) {
-            set_single_persistent_default_layer(_QWERTY);
+    if (record->event.pressed) {
+        switch (keycode) {
+            case COLEMAK:
+                set_single_persistent_default_layer(_COLEMAK);
+                break;
+            case QWERTY:
+                set_single_persistent_default_layer(_QWERTY);
+                break;
+#           if defined(RGBLIGHT_ENABLE) || defined(RGB_MATRIX_ENABLE)
+            case RGBRST:
+#               if defined(RGBLIGHT_ENABLE)
+                eeconfig_update_rgblight_default();
+                rgblight_enable();
+#               else 
+                eeconfig_update_rgb_matrix_default();
+#               endif
+                break;
+#           endif
+            case VRSN:
+                send_string("# qmk " QMK_VERSION "\n");
+                wait_ms(VER_NEWLINE_WAIT);
```

Comment:
just a heads up, you can add a wait to sendstring directly. `SS_DELAY(x)`, IIRC.

Reply:
nice!  Thank you!